app/lib/screens/faq/Item.dart

```diff
@@ -0,0 +1,30 @@
+// ignore_for_file: prefer_const_constructors
+
+import 'package:effektio/common/store/Colors.dart';
+import 'package:effektio/common/widget/FaqListItem.dart';
+
+import 'package:effektio_flutter_sdk/effektio_flutter_sdk.dart';
+import 'package:effektio_flutter_sdk/effektio_flutter_sdk_ffi.dart';
+import 'package:flutter/material.dart';
+import 'dart:async';
+
+class FaqItemScreen extends StatefulWidget {
+  const FaqItemScreen({Key? key, required this.client, required this.faq})
+      : super(key: key);
+  final Client client;
+  final Faq faq;
+
+  @override
+  _FaItemqScreenState createState() => _FaItemqScreenState();
+}
+
+class _FaItemqScreenState extends State<FaqItemScreen> {
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+        appBar: AppBar(
+          title: Text(widget.faq.title()),
+        ),
+        body: Center(child: Text(widget.faq.body())));
+  }
```

Comment:
no back, no sidebar? What does this screen actual look like? Please provide a screenshot.

Reply:
![image](https://user-images.githubusercontent.com/65702289/164234994-a92bbc68-2a50-4993-9b27-5e4496eac5ff.png)

Reply:
that image does not show the screen of the code we are talking about.